### PR TITLE
chore: remove public access modifier in botframework-streaming

### PR DESCRIPTION
**Pull request by stevengum**
Part 2 of #3518
#minor

Description:
Removed public access modifier from botframework-streaming. botframework-streaming is now eslint compliant.

### CODE REVIEW

File: libraries/botframework-streaming/src/assemblers/payloadAssembler.ts [hunk 1]
```diff
@@ -8,32 +8,36 @@
...
 import { StreamManager, PayloadTypes } from '../payloads';
 import { ContentStream } from '../contentStream';
-import { IAssemblerParams } from '../interfaces/IAssemblerParams';
-import { IHeader } from '../interfaces/IHeader';
-import { IResponsePayload } from '../interfaces/IResponsePayload';
-import { IReceiveResponse, IReceiveRequest } from '../interfaces';
-import { IRequestPayload } from '../interfaces/IRequestPayload';
+import { IHeader, IResponsePayload, IReceiveResponse, IReceiveRequest, IRequestPayload } from '../interfaces';
+
+interface AssemblerParams {
```
Comment (joshgummersall) [inline libraries/botframework-streaming/src/assemblers/payloadAssembler.ts:13]: What's the reasoning behind introducing this over `IAssemblerParams`?

File: libraries/botframework-streaming/src/disassemblers/requestDisassembler.ts [hunk 2]
```diff
@@ -5,38 +5,38 @@
...
      * Gets the stream this disassembler is operating on.
+     *
      * @returns An [IStreamWrapper](xref:botframework-streaming.IStreamWrapper) with a Subscribable Stream.
      */
-    public async getStream(): Promise<IStreamWrapper> {
-        const payload: IRequestPayload = { verb: this.request.verb, path: this.request.path, streams: [] };
-        if (this.request.streams) {
+    async getStream(): Promise<IStreamWrapper> {
+        const payload: IRequestPayload = { verb: this.request?.verb, path: this.request?.path, streams: [] };
+        if (this.request?.streams) {
```
Comment (joshgummersall) [inline libraries/botframework-streaming/src/disassemblers/requestDisassembler.ts:39]: You _might_ be able to simplify further as `this.request?.streams?.forEach(...)` without the surrounding `if`.

File: libraries/botframework-streaming/src/httpContentStream.ts [hunk 3]
```diff
@@ -5,30 +5,28 @@
...
      */
-    public constructor(content: HttpContent) {
+    constructor(readonly content: HttpContent) {
         this.id = generateGuid();
-        this.content = content;
         this.description = {
             id: this.id,
-            type: this.content.headers ? this.content.headers.type : 'unknown',
-            length: this.content.headers ? this.content.headers.contentLength : 0,
+            type: this.content?.headers ? this.content?.headers.type : 'unknown',
```
Comment (joshgummersall) [inline libraries/botframework-streaming/src/httpContentStream.ts:28]: `this.content?.headers?.type ?? 'unknown'` and below

File: libraries/botframework-streaming/src/webSocket/webSocketTransport.ts [hunk 4]
```diff
@@ -47,10 +44,11 @@ export class WebSocketTransport implements ITransportSender, ITransportReceiver
      * Sends the given buffer out over the socket's connection.
      *
      * @param buffer The buffered data to send out over the connection.
+     * @returns A number indicating the length of the sent data if the data was successfully sent, otherwise 0.
      */
-    public send(buffer: INodeBuffer): number {
-        if (this._socket && this._socket.isConnected) {
-            this._socket.write(buffer);
+    send(buffer: INodeBuffer): number {
+        if (this.ws && this.ws.isConnected) {
```
Comment (joshgummersall) [inline libraries/botframework-streaming/src/webSocket/webSocketTransport.ts:50]: `this.ws?.isConnected`

**Review by joshgummersall — COMMENTED** [review]
Couple small comments, otherwise LGTM!

File: libraries/botframework-streaming/src/assemblers/payloadAssembler.ts (hunk 1 above)
Comment (stevengum) [reply libraries/botframework-streaming/src/assemblers/payloadAssembler.ts]: I don't remember my rationale... I've removed `AssemblerParams` and applied its stricter `onCompleted` type to `IAssemblerParams.onCompleted`. 😐

**Comment by stevengum** [comment]
@joshgummersall thanks for the review! This PR is ready, although there are some public surface area changes that I'll mull over later today/tomorrow...

**Review by joshgummersall — APPROVED** [review]

**Pull request merged**